src/components/views/rooms/PinnedMessageBanner.tsx

```diff
@@ -90,13 +92,18 @@ export function PinnedMessageBanner({
 
     return (
         <div
+            role="region"
             className="mx_PinnedMessageBanner"
             data-single-message={isSinglePinnedEvent}
             aria-label={_t("room|pinned_message_banner|description")}
             data-testid="pinned-message-banner"
         >
             <button
-                aria-label={_t("room|pinned_message_banner|go_to_message")}
+                aria-description={
```

Comment:
This is a draft attribute hopefully coming in aria 1.3, it is not generally available yet. See note at the top of https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-description